src/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -31,16 +31,35 @@ pscanrules.cachecontrolscanner.soln = Whenever possible ensure the cache-control
 pscanrules.cachecontrolscanner.refs = https://www.owasp.org/index.php/Session_Management_Cheat_Sheet#Web_Content_Caching
 
 pscanrules.contenttypemissingscanner.name = Content-Type Header Missing
+pscanrules.contenttypemissingscanner.name.empty = Content-Type Header Empty
+pscanrules.contenttypemissingscanner.desc = The Content-Type header was either missing or empty.
+contenttypemissingscanner.soln=Ensure each page is setting the specific and appropriate content-type value for the content being delivered.
+contenttypemissingscanner.refs=http://msdn.microsoft.com/en-us/library/ie/gg622941%28v=vs.85%29.aspx
 
 pscanrules.cookiehttponlyscanner.name = Cookie No HttpOnly Flag
+pscanrules.cookiehttponlyscanner.desc=A cookie has been set without the HttpOnly flag, which means that the cookie can be accessed by JavaScript. If a malicious script can be run on this page then the cookie will be accessible and can be transmitted to another site. If this is a session cookie then session hijacking may be possible.
+pscanrules.cookiehttponlyscanner.soln=Ensure that the HttpOnly flag is set for all cookies.
+pscanrules.cookiehttponlyscanner.refs=http://www.owasp.org/index.php/HttpOnly
 
 pscanrules.cookiesecureflagscanner.name = Cookie Without Secure Flag
+pscanrules.cookiesecureflagscanner.desc=A cookie has been set without the secure flag, which means that the cookie can be accessed via unencrypted connections.
+pscanrules.cookiesecureflagscanner.soln=Whenever a cookie contains sensitive information or is a session token, then it should always be passed using an encrypted channel. Ensure that the secure flag is set for cookies containing such sensitive information.
+pscanrules.cookiesecureflagscanner.refs=http://www.owasp.org/index.php/Testing_for_cookies_attributes_(OWASP-SM-002)
 
 pscanrules.crossdomainscriptinclusionscanner.name = Cross-Domain JavaScript Source File Inclusion
+pscanrules.crossdomainscriptinclusionscanner.desc=The page includes one or more script files from a third-party domain.
+pscanrules.crossdomainscriptinclusionscanner.soln=Ensure JavaScript source files are loaded from only trusted sources, and the sources can't be controlled by end users of the application.
 
 pscanrules.mixedcontentscanner.name = Secure Pages Include Mixed Content
+pscanrules.mixedcontentscanner.name.inclscripts = Secure Pages Include Mixed Content (Including Scripts)
+pscanrules.mixedcontentscanner.desc=The page includes mixed content, that is content accessed via HTTP instead of HTTPS.
+pscanrules.mixedcontentscanner.soln=A page that is available over SSL\TLS must be comprised completely of content which is transmitted over SSL\TLS.\nThe page must not contain any content that is transmitted over unencrypted HTTP.\n This includes content from third party sites.
```

Comment:
Better use `/` for SSL TLS separation (otherwise the `/` needs to be escaped to be shown).